### ISSUE TITLE
Leading Zero check throwing false positives

### ISSUE THREAD

**Issue by rossPatton** [rossPatton/stylint#25]
width 40.9473684211%

should be considered valid

**Comment by leny** [comment]
Hi, this bug still exists. Maybe reopen?

**Comment by rossPatton** [comment]
hi, can you please add your config (if using one) and version #? i'll

**Comment by leny** [comment]
Stylint version: `1.3.2`

Config: ```
{
    "blocks": false,
    "brackets": {
        "expect": "never",
        "error": true
    },
    "colons": {
        "expect": "never",
        "error": true
    },
    "colors": false,
    "commaSpace": {
        "expect": "always",
        "error": true
    },
    "commentSpace": {
        "expect": "always",
        "error": true
    },
    "cssLiteral": {
        "expect": "never",
        "error": true
    },
    "depthLimit": {
        "expect": 4,
        "error": true
    },
    "duplicates": {
        "expect": true,
        "error": false
    },
    "efficient": {
        "expect": "always",
        "error": true
    },
    "extendPref": false,
    "globalDupe": false,
    "indentPref": 4,
    "leadingZero": {
        "expect": "never",
        "error": false
    },
    "maxErrors": false,
    "maxWarnings": false,
    "mixed": false,
    "namingConvention": {
        "expect": "lowercase-dash",
        "error": true
    },
    "namingConventionStrict": {
        "expect": true,
        "error": true
    },
    "none": {
        "expect": "never",
        "error": true
    },
    "noImportant": {
        "expect": true,
        "error": false
    },
    "parenSpace": {
        "expect": "always",
        "error": true
    },
    "placeholders": false,
    "prefixVarsWithDollar": {
        "expect": "never",
        "error": false
    },
    "quotePref": {
        "expect": "double",
        "error": true
    },
    "semicolons": {
        "expect": "never",
        "error": true
    },
    "sortOrder": false,
    "stackedProperties": {
        "expect": "never",
        "error": true
    },
    "trailingWhitespace": {
        "expect": false,
        "error": true
    },
    "universal": {
        "expect": "never",
        "error": true
    },
    "valid": {
        "expect": true,
        "error": false
    },
    "zeroUnits": {
        "expect": "never",
        "error": true
    },
    "zIndexNormalize": {
        "expect": 5,
        "error": true
    }
}
```

The bug occurs on line like : 

```
height: 10.5rem
margin: 120.5rem
```

**Comment by rossPatton** [comment]
thx, will look into it
